VideoLocker/src/main/java/org/edx/mobile/view/adapters/InfiniteScrollUtils.java

```diff
@@ -124,14 +125,31 @@ public void loadMore() {
         }
 
         private void onLoadMore() {
+            onLoadMore(false);
+        }
+
+        /**
+         * Variant of the {@link #onLoadMore} function allowing us to control the visibility of
+         * progress and lazily clear adapter after a page has loaded.
+         *
+         * @param lazyClearanceEnabled <code>true</code> if we want to clear the adapter after the
+         *                             page has loaded without showing progress, otherwise
+         *                             <code>false</code>.
```

Comment:
Alternative name and description, that is possibly more concise and clear as to the intent:

````java
/**
  * @param isRefreshingSilently <code>true</code> If we're doing a silent refresh,
  *                             <code>false</code> if pagination is being done as usual.
  */
````